lbrynet/core/utils.py

```diff
@@ -9,6 +9,8 @@
 
 import pkg_resources
 
+from twisted.internet import reactor
```

Comment:
The worry is what happen if you don't want the default reactor:

"IMPORTANT: installing a reactor should be the first thing done in the app, since any code that does from twisted.internet import reactor will automatically install the default reactor if the code hasn’t already installed one." (from http://twistedmatrix.com/documents/current/core/howto/choosing-reactor.html)

And, as Jack referenced, this is largely a problem if the lbrynet library is being using in a gui app that needs to configure the reactor to use the GUI's event loop.  It might also be an issue for testing, but I think `trial` should take care of it.

Regardless, I think importing reactor at module load-time makes it harder to use the library.